packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -262,8 +265,37 @@ export default function CollabSidebar() {
 		};
 	}, [] );
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType, {
+		id: postId,
```

Comment:
Can there be a case where there is no current post ID?

Reply:
I think it does, Ill put condition there to avoid errors.

Reply:
After some finding, Undefined postId is already being handled under `useEntityBlockEditor` for cases like templates and patterns.